api/api-default-job-timeout-v1/src/main/java/com/thoughtworks/go/apiv1/defaultjobtimeout/representers/DefaultJobTimeOutRepresenter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.defaultjobtimeout.representers;
+
+import com.thoughtworks.go.api.base.OutputWriter;
+import com.thoughtworks.go.api.representers.JsonReader;
+import com.thoughtworks.go.spark.Routes;
+
+import static com.thoughtworks.go.CurrentGoCDVersion.apiDocsUrl;
+
+public class DefaultJobTimeOutRepresenter {
+    public static void toJSON(OutputWriter outputWriter, String defaultJobTimeout) {
+        outputWriter.addLinks(outputLinkWriter -> outputLinkWriter
+                .addAbsoluteLink("doc", apiDocsUrl("#default-job-timeout"))
+                .addLink("self", Routes.DefaultJobTimeout.BASE))
+                .add("default_job_timeout", defaultJobTimeout);
+    }
+
+    public static String fromJson(JsonReader jsonReader) {
+        return jsonReader.getStringOrDefault("default_job_timeout", null);
```

Comment:
The above line will set the `default_job_timeout` to null if the user has not specified it in the request body.
IMO, Server should throw an error if the `default_job_timeout` field is not specified in the request body.

Use `jsonReader.getString` instead?